### PR TITLE
api: add hardcoded versioning support

### PR DESCRIPTION
**Pull request by GRISHNOV**
Add versioning support, now the module api has `_VERSION` hardcoded variable. Is part of the task [1].

1. https://github.com/tarantool/roadmap-internal/issues/204

### CODE REVIEW

File: kafka/init.lua [hunk 1]
```diff
@@ -471,4 +471,5 @@ return {
     Consumer = Consumer,
     Producer = Producer,
     _LIBRDKAFKA = tnt_kafka.librdkafka_version(),
+    _VERSION = require('kafka.version'),
```
Comment (DifferentialOrange) [inline kafka/init.lua:474]: You're missing `kafka/CMakeLists.txt` update
```bash
tt rocks STATIC_BUILD=ON make

tree .rocks/
.rocks/
├── lib
│   └── tarantool
│       └── kafka
│           └── tntkafka.so
└── share
    └── tarantool
        ├── kafka
        │   └── init.lua
        └── rocks
            ├── kafka
            │   └── scm-1
            │       ├── doc
            │       │   ├── LICENSE
            │       │   └── README.md
            │       ├── kafka-scm-1.rockspec
            │       └── rock_manifest
            └── manifest

10 directories, 7 files
```

Comment (GRISHNOV) [reply kafka/init.lua:474]: Fixed

File: .github/workflows/fast_testing.yml [hunk 2]
```diff
@@ -6,6 +6,19 @@ on:
...
+    if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Check module version
+        uses: tarantool/actions/check-module-version@igrishnov/gh-204-extension-check-module-version
+        with:
+          module-name: 'kafka'
+          rock-make-opts: 'STATIC_BUILD=ON'
+
+
```
Comment (DifferentialOrange) [inline .github/workflows/fast_testing.yml:21]: excessive newline

**Review by DifferentialOrange — COMMENTED** [review]
Seems fine, let's wait for the patch merge

**Review by DifferentialOrange — APPROVED** [review]

**Pull request merged**